### ISSUE TITLE
nnf_group_norm causing R to crash

### ISSUE THREAD

**Issue by Athospd** [mlverse/torch#473]
`nnf_group_norm()` is causing R to crash. On the code below, we expect `nnf_group_norm(tt, 3)` and `nnf_layer_norm(tt, c(3,3))` to return the same output.

``` r
library(torch)
tt <- torch_tensor(t(matrix(1:3, 3, 3)) * 1.)
nnf_group_norm(tt, 3) # <------ CRASHES (do not run)

nnf_layer_norm(tt, c(3,3))
#> torch_tensor
#> -1.2247  0.0000  1.2247
#> -1.2247  0.0000  1.2247
#> -1.2247  0.0000  1.2247
#> [ CPUFloatType{3,3} ]
```

<sup>Created on 2021-02-15 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

Python version

![image](https://user-images.githubusercontent.com/1925102/108002830-90039780-6fcf-11eb-8bd7-48ebcfa4d920.png)

``` r
library(torch)
sessionInfo()
#> R version 4.0.2 (2020-06-22)
#> Platform: x86_64-pc-linux-gnu (64-bit)
#> Running under: Ubuntu 20.04.1 LTS
#> 
#> Matrix products: default
#> BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.9.0
#> LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.9.0
#> 
#> locale:
#>  [1] LC_CTYPE=pt_BR.UTF-8       LC_NUMERIC=C              
#>  [3] LC_TIME=pt_BR.UTF-8        LC_COLLATE=en_US.UTF-8    
#>  [5] LC_MONETARY=pt_BR.UTF-8    LC_MESSAGES=en_US.UTF-8   
#>  [7] LC_PAPER=pt_BR.UTF-8       LC_NAME=C                 
#>  [9] LC_ADDRESS=C               LC_TELEPHONE=C            
#> [11] LC_MEASUREMENT=pt_BR.UTF-8 LC_IDENTIFICATION=C       
#> 
#> attached base packages:
#> [1] stats     graphics  grDevices utils     datasets  methods   base     
#> 
#> other attached packages:
#> [1] torch_0.2.1.9000
#> 
#> loaded via a namespace (and not attached):
#>  [1] Rcpp_1.0.5      ps_1.5.0        digest_0.6.27   magrittr_2.0.1 
#>  [5] coro_1.0.1      evaluate_0.14   highr_0.8       rlang_0.4.10   
#>  [9] stringi_1.5.3   callr_3.5.1     rmarkdown_2.6   tools_4.0.2    
#> [13] stringr_1.4.0   bit64_4.0.5     bit_4.0.4       processx_3.4.5 
#> [17] xfun_0.20       yaml_2.2.1      compiler_4.0.2  htmltools_0.5.1
#> [21] knitr_1.30
```

<sup>Created on 2021-02-15 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by Athospd** [comment]
I traced the error and I found out that `cudnn_enable` argument from `torch:::torch_group_norm` is causing this problem.

```
library(torch)
tt <- torch_tensor(t(matrix(1:3, 3, 3)) * 1.)

debugonce(torch:::torch_layer_norm) # <---------- cudnn_enable is FALSE
nnf_layer_norm(tt, c(3,3))

debugonce(torch:::torch_group_norm) # <---------- cudnn_enable is a Tensor (same as input)
nnf_group_norm(tt, 1) 
```

Probable problem: commented arg on `nnf_group_norm`

![image](https://user-images.githubusercontent.com/1925102/108101077-862c7380-7065-11eb-85d4-fdfdb132fad4.png)

**Comment by dfalbel** [comment]
Thanks! Fixed in #474
